sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -815,7 +815,7 @@ class SubquerySuite extends QueryTest with SharedSQLContext {
 
   // Generate operator
   test("Correlated subqueries in LATERAL VIEW") {
-    withTempView("t1", "t2") {
+    withTempView("t1", "t2", "t3") {
```

Comment:
uh... i took out the usage of t3 and forgot to take out from this line. Thanks for catching this.